bundles/org.eclipse.orion.client.javascript/web/javascript/hover.js

```diff
@@ -436,8 +436,99 @@ Objects.mixin(JavaScriptHover.prototype, /** @lends javascript.JavaScriptHover.p
     }
 });
 
+/**
+ * @name javascript.JavaScriptDebugHoverEvaluationProvider
+ * @constructor
+ * @public
+ * @param {javascript.ASTManager} astManager
+ * @param {javascript.ScriptResolver} resolver
+ * @param {javascript.TernWorkerCore} ternWorker
+ * @param {javascript.CUProvider} cuProvider
+ */
+function JavaScriptDebugHoverEvaluationProvider(astManager, resolver, ternWorker, cuProvider) {
```

Comment:
JavaScriptDebugHover would be better, but thats fine for now, we can change it later if we really don't like it